src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,40 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private K[] keys = (K[]) new Object[MAX_SIZE];
+    private V[] values = (V[]) new Object[MAX_SIZE];
+    private int size = 0;
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if ((key == keys[i] || key != null && key.equals(keys[i]))) {
+                keys[i] = key;
+                values[i] = value;
+                return;
+            }
+        }
+        size++;
+        keys[size - 1] = key;
+        values[size - 1] = value;
```

Comment:
```suggestion
        keys[size] = key;
        values[size] = value;
        size++;
```
там ви зробимо на 2 операції менше)